### PR TITLE
Add Barbie Seahorse Adventures

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/dulsi/seahorse-adventures/issues/1
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/dulsi/seahorse-adventures/pull/2

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.imitationpickles.barbie

**Comment by Mailaender** [comment]
bot, build org.imitationpickles.barbie

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/org.imitationpickles.barbie.appdata.xml: FAILED:
• tag-missing           : <release> required
Validation of files failed
```

File: org.imitationpickles.barbie.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+
+modules:
+  - name: portmidi
+    buildsystem: cmake
```
Comment (hfiguiere) [inline org.imitationpickles.barbie.yaml:18]:
```suggestion
    buildsystem: cmake-ninja
```
?

Comment (Mailaender) [reply org.imitationpickles.barbie.yaml]: Why not.

**Comment by Mailaender** [comment]
bot, build org.imitationpickles.barbie

**Comment by Highbeam62** [comment]
STOP!!? ENTIRELY TOO MANY EMAILS. LITERALLY 25 BEFORE NOON!!!!!!

**Comment by barthalion** [comment]
"Barbie" sounds legally dodgy, but let me see first what the merge command complains about.

/merge

**Comment by barthalion** [comment]
bot, build org.imitationpickles.barbie

**Review by Mailaender — COMMENTED** [review]
I didn't use any shared modules.

**Comment by barthalion** [comment]
Then you should have removed `.gitmodules` file instead of removing only the directory.

**Pull request closed without merging**